src/tree/utils/hasComplexChild.ts

```diff
@@ -0,0 +1,8 @@
+import { isRegularNode, RegularNode } from '@stoplight/json-schema-tree';
+
+export function hasNoComplexChild(
+  node: RegularNode,
+): node is Omit<RegularNode, 'children'> & { children: (RegularNode & { simple: true })[] } {
+  // eslint-disable-next-line @typescript-eslint/prefer-optional-chain
```

Comment:
![image](https://user-images.githubusercontent.com/9273484/102420715-094c3180-4003-11eb-95ef-bfbddcfd8406.png)
![image](https://user-images.githubusercontent.com/9273484/102420719-0c472200-4003-11eb-9fb8-6f7008674167.png)

:roll_eyes:

Reply:
Luckily, this file is no longer needed, haha.
I noticed I posted the same screenshots, but the problem was that if I applied one suggestion, the other rule kicked in and started yelling at me.